shopBuddy/shopBuddy/GlobalManager.h

```diff
@@ -0,0 +1,25 @@
+//
+//  GlobalManager.h
+//  shopBuddy
+//
+//  Created by Rachna Gupta on 7/27/22.
+//
+
+#import <Foundation/Foundation.h>
+#import "Item.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface GlobalManager : NSObject
+
+@property (atomic, strong) NSMutableDictionary<NSString *, NSString *> *itemJobIdMap;
+@property (atomic, strong) NSMutableArray<NSString *> *outstandingJobs; // Job sync in progress
+@property (atomic, strong) NSMutableDictionary<NSString *, NSDictionary *> *completeJobs;
+
++ (id)sharedManager;
+
+- (void)fetchPricesWithItem:(Item *)item fromStore: (NSString *)store completion:(void(^)(NSDictionary *prices, BOOL success))completion;
```

Comment:
Right now, the types of the results are all over the place, because I haven't parsed them yet. I'll implement this in the next PR when it's cleanly just NSString to NSString.